packages/getters/src/balances-response.ts

```diff
@@ -7,10 +7,7 @@ export const getBalanceView = createGetter(
   (balancesResponse?: BalancesResponse) => balancesResponse?.balanceView,
 );
 
-export const getAssetIdFromBalancesResponseOptional = getBalanceView
-  .optional()
-  .pipe(getMetadata)
-  .pipe(getAssetId);
+export const getAssetIdFromBalancesResponse = getBalanceView.pipe(getMetadata).pipe(getAssetId);
```

Comment:
We can now just call `.optional()` on this, so no need to hard-code an optional version.